### DISCUSSION TITLE
Feature preview: custom spectrograms

### DISCUSSION THREAD

**Discussion by Mattk70** [Mattk70/Chirpity-Electron#119] [Show and tell]
I've often wondered how to make Chirpity output "clean" spectrograms, like those you see produced by Magnus Robb, along with his colleagues on the Sound Approach website. Now, after some tinkering, I think I have a solution. With custom spectrograms, you will be able to set the colours for the high and low amplitude sounds. Using the Peak Threshold slider, you can adjust the spectrogram to "remove" sounds below a threshold from the display. Here's an illustration, one which gives a Spectrogram like those you see on the Xeno-Canto website:

https://github.com/Mattk70/Chirpity-Electron/assets/61826357/db51a1e9-5d32-4e05-a9c8-82e8ba3ee0c9

**Comment by Vollpflock** [comment]
Can't wait for that. Next up: noise reduction, and I'll be on cloud nine.

**Comment by Mattk70** [comment]
Ha! If you know a setting in Audacity for NR that's effective, I'd love to know about it! I find any setting that has a resonable impact on noise also generates terrible robot-sounding artefacts.
